symfony/src/Twig/AppExtension.php

```diff
@@ -22,7 +29,10 @@ public function getFunctions()
             new TwigFunction('path_exists', [$this, 'checkPath']),
             new TwigFunction('asset', [$this, 'asset']),
             new TwigFunction('slugify', [$this, 'slugify']),
-            new TwigFunction('timezone_display', [$this, 'timezoneDisplay'])
+            new TwigFunction('timezone_display', [$this, 'timezoneDisplay']),
+            new TwigFunction('codebook_display', [$this, 'getCodeBookDisplay']),
+            new TwigFunction('organization_display', [$this, 'getAwardeeDisplay']),
+            new TwigFunction('awardee_display', [$this, 'getAwardeeDisplay'])
```

Comment:
The intent is to replicate what we were doing in `HpoApplication`. Both `getAwardeeDisplay` and `getOrganizationDisplay` likely need some level of caching in case the lookup happens in a loop.

Reply:
Opted to cache it in memory.